lib/models/CartoonCat.js

```diff
@@ -1,4 +1,26 @@
 const pool = require('../utils/pool');
 module.exports = class CartoonCat {
   // TODO -- fill in this class!
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.lives = row.lives;
+    this.url = row.url;
+    this.year = row.year;
+    this.isSidekick = row.is_sidekick;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * from cartoon_cats');
+    return rows.map((catRow) => new CartoonCat(catRow));
+  }
+
+  static async getById(id) {
+    const { rows } = await pool.query(
+      'Select * from cartoon_cats WHERE id = $1',
```

Comment:
so, it really doesn't matter, but the convention is to write your SQL in all caps so it's easier to identify table names / column names. but either way, it should be consistent (all caps or no caps) (no cap) so this would look like either:
```SELECT * FROM cartoon_cats WHERE id = $1```
or
```select * from cartoon_cats where id = $1```